interface/main/tabs/js/tabs_view_model.js

```diff
@@ -19,6 +19,11 @@ function tabStatus(title,url,name,closable,visible,locked)
     self.title=ko.observable(title);
     self.url=ko.observable(url);
     self.name=ko.observable(name);
+    self.loading_text=ko.observable((xl("Loading") + " " + label + "..."));
```

Comment:
@sjpadgett I was unable to understand why to consider html `xl("Loading")` and `label` both will return text

Reply:
oops, you're right. Sorry. you've got spinner all over my head now sooooo. carry on:)